code/modules/modular_computers/NTNet/NTNRC/conversation.dm

```diff
@@ -1,35 +1,46 @@
 #define MAX_CHANNELS 1000
 
 /datum/ntnet_conversation
-	var/id = null
+	///The title of the conversation, seen in the UI.
 	var/title = "Untitled Conversation"
-	var/datum/computer_file/program/chatclient/operator // "Administrator" of this channel. Creator starts as channel's operator,
-	var/list/messages = list()
-	///chat clients who are active or minimized
-	var/list/active_clients = list()
-	///chat clients who have exited out of the program.
-	var/list/offline_clients = list()
-	///clients muted by operator
-	var/list/muted_clients = list()
-	//if a channel is strong, it cannot be renamed or deleted.
+	//If a channel is strong, it cannot be renamed or deleted.
 	var/strong = FALSE
+	///The password to join a channel, set by an Administrator.
 	var/password
+
+	///A static UID to ensure no conversations are the same.
 	var/static/ntnrc_uid = 0
+	///ID using the UID.
+	var/id
+
+	///List of all messages sent in the conversation.
+	var/list/messages = list()
+
+	///The "Administrator" of the channel, the creator starts as channel's operator by default.
+	var/datum/computer_file/program/chatclient/channel_operator
+	///Chat clients who are active or minimized.
+	var/list/datum/computer_file/program/chatclient/active_clients = list()
+	///Chat clients who have exited out of the program.
+	var/list/datum/computer_file/program/chatclient/offline_clients = list()
+	///Chat clients currently muted by the operator, rendering them unable to ping other people.
+	var/list/datum/computer_file/program/chatclient/muted_clients = list()
 
-/datum/ntnet_conversation/New()
-	id = ntnrc_uid + 1
+/datum/ntnet_conversation/New(title, strong = FALSE)
+	src.title = title
+	src.strong = strong
+
+	id = ntnrc_uid++
```

Comment:
Don't do this please, it's confusing if you don't understand the rules and does not make the code cleaner

Reply:
```suggestion
	id = ntnrc_uid
	ntnrc_uid++
```